packages/components/src/color-palette/test/__snapshots__/index.js.snap

```diff
@@ -321,9 +321,6 @@ exports[`ColorPalette should render a dynamic toolbar of colors 1`] = `
           <Icon
             icon={
               <SVG
-                aria-hidden={true}
```

Comment:
As noted in the other branch, those three properties are necessary for any SVG used as an icon, and they need to be added as attributes to the SVG itself. Whether they be baked into the SVG itself or added runtime or in some other fashion, they have to be there.

Reply:
They are already there, they're added by the SVG primitive.

Reply:
Oh apologies I missed the full understanding of the "primitive" aspect here. So `SVG` adds it, gotcha, 👍 👍 blaze a trail!